src/gen_rpc_client.erl

```diff
@@ -465,14 +473,15 @@ send_ping(#state{socket=Socket, driver=Driver, driver_mod=DriverMod} = State) ->
             {noreply, State, gen_rpc_helper:get_inactivity_timeout(?MODULE)}
     end.
 
-cast_worker(NodeOrTuple, Cast, Ret, SendTO) ->
+cast_worker(NodeOrTuple, Cast, Ret, SendTimeout) ->
     %% Create a unique name for the client because we register as such
     PidName = ?NAME(NodeOrTuple),
-    ?tp(gen_rpc_cast, #{ cast => Cast
-                       , target => NodeOrTuple
-                       , sendto => SendTO
-                       , pid => PidName
-                       }),
+    ?tp(gen_rpc_input, #{ type => cast
```

Comment:
Nit: the type may be `oc` as well, right?  Since this function is also called [here](https://github.com/emqx/gen_rpc/pull/22/files#diff-039b498b6deb12b31dbad0d77be33506a347cdf3497f089bb1f9bad59fe9124cR147)?

Reply:
Removed the confusing field.